packages/synapse-react-client/src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -27,26 +26,29 @@
       SRC.$bootstrap-4-danger-text-color
     );
   }
+
+  // PORTALS-2597, SWC-6413: all buttons should be rounded
+  button {
+    border-radius: SRC.$button-border-radius;
+    text-transform: SRC.$button-text-transform;
+  }
```

Comment:
Apply button styling even when .boostrap-4-backport overrides MUI theme styling, see "Date Posted" button in DiscussionThread:
main | feature
:---:|:---:
<img width="1293" alt="main_DiscussionThread" src="https://user-images.githubusercontent.com/26949006/231309802-70569dc4-f794-45d2-ab41-287d80ba0bbc.png"> | <img width="1291" alt="feature_DiscussionThread" src="https://user-images.githubusercontent.com/26949006/231309818-fb1aba48-2616-45a5-b3a9-9906d85bebdb.png">

Reply:
Would rather we get rid of `bootstrap-4-backport` but this works for now!